<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>653</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>134</red>
        <green>153</green>
        <blue>181</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>134</red>
        <green>153</green>
        <blue>181</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>134</red>
        <green>153</green>
        <blue>181</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>134</red>
        <green>153</green>
        <blue>181</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MyOwnMessenger</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QPushButton {
	background-color: #39424f;
	border: 1px solid #39424f;
	border-radius: 4px;
	color: #fafafa;

}
QPushButton:hover {
	background-color: #BBC7DA;
}
QPushButton:focus {
	outline: none;
	border: 1px solid #8699B5;
	text-decoration: underline;
}
QPushButton:pressed {
	background-color: #8699B5;
}

QListWidget {
	border: none;
}
QListWidget QWidget {
	margin: 0;
	padding: 0;
}

QListWidget QPushButton {
	width: 16px;
	height: 16px;
	margin: 0;
	background-color: #fff;
	border: none;
	color: orange;
}
QListWidget QPushButton:hover {
	background-color: orange;
	color: white;
}

QTextEdit {
		border: 1px solid #8699B5;
	border-radius: 4px;
}
QTextEdit:focus {
	border: 3px solid #8699B5;
}

QListView::item:alternate {
    background: #EEEEEE;
}

QListView::item:selected {
    border: 1px solid #8699B5;
	color: #000;
	font-weight: 800;
	background-color: #fff;
}

QListView::item:selected:!active {
    border: 1px solid lightgrey;
}

QListView::item:hover {
    border: 1px solid lightgrey;
}

QLineEdit {
	border: 1px solid #66BB6A;
	border-radius: 4px;
}

QLineEdit[echoMode=&quot;2&quot;] {
    lineedit-password-character: 9679;
}

QLineEdit:focus {
	border: 2px solid #8699B5;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>400</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Change contact list</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QTextEdit" name="textEditUsername">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddContact">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../Downloads/icons8-plus.png</normaloff>../../../../../Downloads/icons8-plus.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonGroupChat">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Chat</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>151</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Contact list</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QListWidget" name="listWidgetContacts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>391</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Current chat</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QTextEdit" name="textEditChat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>371</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>391</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Enter your message</string>
    </property>
    <widget class="QTextEdit" name="textEditMessage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>b.jpg</normaloff>b.jpg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonItalics">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>i.jpg</normaloff>i.jpg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonUnderline">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>u.jpg</normaloff>u.jpg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSmile">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>ab.gif</normaloff>ab.gif</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMelancholy">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>ac.gif</normaloff>ac.gif</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSurprise">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>ai.gif</normaloff>ai.gif</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>530</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="PushButtonSend">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icon.ico</normaloff>icon.ico</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>391</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>History</string>
    </property>
    <widget class="QTextEdit" name="textEditHistory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>28</y>
       <width>371</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelAvatar">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>generic-avatar.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonChangeAvatar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Avatar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>653</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
